opta/constants.py

```diff
@@ -42,7 +42,7 @@
 OPTA_DISABLE_REPORTING = "OPTA_DISABLE_REPORTING"
 TF_FILE_PATH = "main.tf.json"
 TF_PLAN_PATH = "tf.plan"
-MAX_TERRAFORM_VERSION = "1.1.0"
+# MAX_TERRAFORM_VERSION = "1.1.0"
```

Comment:
Should we proactively not support v2 (would be the next major version)?

Reply:
A good point made...we should do that... let me make this `2.0.0`. That should suffice I guess?

Reply:
Yes